_commands/os_x/screencapture.md

```diff
@@ -0,0 +1,88 @@
+---
+---
+
+screencapture
+-------
+
+`screencapture` is used to capture an image of the whole, or part of the screen.
+
+~~~ bash
+#Takes a screenshot of the entire screen and saves it to the file screencap.png in the pwd
```

Comment:
`pwd` stands for "print working directory". Saying 'cwd' would be more correct here imho.